src/main/java/com/google/devtools/build/lib/bazel/rules/BazelRuleClassProvider.java

```diff
@@ -111,7 +111,7 @@
   public static class StrictActionEnvOptions extends FragmentOptions {
     @Option(
         name = "experimental_strict_action_env",
```

Comment:
I think we just want this flag to go away. If you want something like the old behavior, the correct thing to do is to pass `--action_env=PATH` and `--action_env=LD_LIBRARY_PATH`. `--noexperimental_strict_action_env` does some universally bad things like reading from the server environment rather than the client environment.

Reply:
You are right. I was under the impression that without --experimental_strict_action_env all shell environment variables are inherited by the action environment but I was wrong there.